solr/solrj-jetty/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -102,7 +102,14 @@
  * HttpJettySolrClient}.
  */
 public class Http2SolrClient extends HttpSolrClientBase {
+  // nocommit rename to HttpJettySolrClient
+
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+  /**
+   * A Java system property to select the {@linkplain HttpClientBuilderFactory} used for configuring
+   * HTTP based SolrClients.
+   */
+  public static final String SYS_PROP_HTTP_CLIENT_BUILDER_FACTORY = "solr.httpclient.builder.factory"; // nocommit
```

Comment:
Should be renamed, like maybe `solr.solrj.jetty.factory`?  (for ref guide as well).